clusterloader2/pkg/util/cluster.go

```diff
@@ -158,3 +158,23 @@ func GetMasterIPs(c clientset.Interface, addressType corev1.NodeAddressType) ([]
 	}
 	return ips, nil
 }
+
+// LegacyIsMasterNode returns true if given node is a registered master according
```

Comment:
Can you move introducing this function and replacing the old usage to a separate PR (that will be merged first)?

It's obviously correct, and with go-modules, we've seen different problems in the past in different places, so the risk of rollback is higher.

Reply:
Oh merge that PR first, I understand now! Thanks